mempool/clist_mempool.go

```diff
@@ -148,7 +139,9 @@ func (mem *cListMempool) recheckTxs(proxyAppConn proxy.AppConnMempool) {
 	proxyAppConn.FlushAsync()
 }
 
-func (mem *cListMempool) getNextTxBytes(remainBytes int64, remainGas int64, starter []byte) ([]byte, error) {
+// GetNextTxBytes finds satisfied tx with two iterations which cost O(N) time, will be optimized with balance tree
+// or other techniques to reduce the time complexity to O(logN) or even O(1)
```

Comment:
Comments can be updated, O(logN) by llrb had been implemented, and O(1) had been found not applicable here.